restorers/dataloader/base/commons.py

```diff
@@ -21,3 +21,10 @@ def random_vertical_flip(low_image, enhanced_image):
             tf.image.flip_up_down(enhanced_image),
         ),
     )
+
+
+def read_image(image_path: str, normalization_factor) -> tf.Tensor:
+    image = tf.io.read_file(image_path)
+    image = tf.io.decode_image(image, channels=3, expand_animations=False)
+    image = tf.cast(image, dtype=tf.float32) / normalization_factor
```

Comment:
Should reading an image contain normalization code in it?

This is not a blocker -- I just wanted to point it out.

Reply:
We are reading images to be fed into the model and hence the `normalization_fector` is kept here. I think its better to set it to `1` by default.

Reply:
That makes more sense to me! Thanks,